cmd/k8s-operator/egress-eps.go

```diff
@@ -142,14 +156,21 @@ func (er *egressEpsReconciler) podIsReadyToRouteTraffic(ctx context.Context, pod
 		l.Debugf("proxy Pod is being deleted, ignore")
 		return false, nil
 	}
-	podIP := pod.Status.PodIP
+	podIP, err := podIPv4(&pod)
+	if err != nil {
+		return false, fmt.Errorf("error determining Pod IP address: %v", err)
+	}
+	if podIP == "" {
+		l.Infof("[unexpected] Pod does not have an IPv4 address, this will most likely not work")
```

Comment:
Given that we're returning early, I think we can be stronger than "most likely" :)

```suggestion
		l.Infof("[unexpected] Pod does not have an IPv4 address, and IPv6 is not currently supported")
```